setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.5.0
```

Comment:
This said 1.5.0 but the tag doesn't exist, and neither does the release, so I set it back to 1.4.2


Let me know if I missed something and we should use `1.5.0`